.gitignore

```diff
@@ -44,3 +44,4 @@ node_modules/
 # IDE specific
 .vscode/
 yarn.lock
+package-lock.json
```

Comment:
https://github.com/gatsbyjs/gatsby/pull/1070

We do commit the root yarn.lock file.https://github.com/gatsbyjs/gatsby/blob/1.0/yarn.lock